config.js

```diff
@@ -67,10 +65,16 @@ System.config({
       "aurelia-pal": "npm:aurelia-pal@1.0.0-beta.1.0.2",
       "core-js": "npm:core-js@1.2.6"
     },
+    "npm:aurelia-dependency-injection@1.0.0-beta.1.0.1": {
```

Comment:
can't be helped (easiliy). aurelia-templating uses 1.0.1 and aurelia-framework 1.0.0.